src/main/java/org/fundacionjala/coding/nestor/Main.java

```diff
@@ -0,0 +1,21 @@
+package org.fundacionjala.coding.nestor;
+
+public class Main {
+    public static void main(String[] args) {
+        Movie terminator = new RegularMovie("Terminator");
+        Movie srek = new ChildrenMovie("Srek");
+        Movie frozen = new ChildrenMovie("Frozen");
+        Movie goldFather = new RegularMovie("The Gold Father");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "goldFather". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "goldFather" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)